src/Servers/IIS/src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -257,6 +258,31 @@ HOSTFXR_UTILITY::GetAbsolutePathToDotnet(
         return dotnetViaWhere.value();
     }
 
+    BOOL isWow64Process = false;
+    if (LOG_LAST_ERROR_IF_NOT(IsWow64Process(GetCurrentProcess(), &isWow64Process)))
```

Comment:
If `IsWow64Process()` returns false, it doesn't mean it is a 64 bit process. It's either a 64 bit process running on a 64 bit machine, or a 32 bit process running on a 32 bit machine. This is the same check: https://github.com/aspnet/AspNetCore/blob/3c09d644cccdb21801f7a79e1188a1a1212de5d9/src/Servers/IIS/src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp#L426-L439. Let's pull this out into a separate method so it can be used in both places.